<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgCell">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MplWidget" name="mplEduHist" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>269</y>
     <width>290</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QLabel" name="lblEduAvg">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mean</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEduMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgEdu">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEduMax">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <zorder>imgEdu</zorder>
   <zorder>lblEduAvg</zorder>
   <zorder>lblEduMin</zorder>
   <zorder>lblEduMax</zorder>
  </widget>
  <widget class="QLabel" name="lblId">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgPericentrin">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imgTubulin">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTubAvg">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>mean</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTubMax">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTubMin">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plotButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>440</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSampleOK">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Is sample ok?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOK">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ok/nok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_nc">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ok/nok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_nb">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ok/nok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_cc">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ok/nok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_cb">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ok/nok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_nb_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nuclei boundary</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_cb_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cell boundary</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_nc_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nuclei centroid</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDist_cc_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cell centroid</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgCentrCloseup">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblCentr_n_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nÂº of centrosomes</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCentr_n">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>mplEduHist</zorder>
  <zorder>prevButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>imgCell</zorder>
  <zorder>lblId</zorder>
  <zorder>imgPericentrin</zorder>
  <zorder>imgTubulin</zorder>
  <zorder>lblTubAvg</zorder>
  <zorder>lblTubMax</zorder>
  <zorder>lblTubMin</zorder>
  <zorder>plotButton</zorder>
  <zorder>lblSampleOK</zorder>
  <zorder>lblOK</zorder>
  <zorder>lblDist_nc</zorder>
  <zorder>lblDist_nb</zorder>
  <zorder>lblDist_cc</zorder>
  <zorder>lblDist_cb</zorder>
  <zorder>lblDist_nb_2</zorder>
  <zorder>lblDist_cb_2</zorder>
  <zorder>lblDist_nc_2</zorder>
  <zorder>lblDist_cc_2</zorder>
  <zorder>imgCentrCloseup</zorder>
  <zorder>lblCentr_n_2</zorder>
  <zorder>lblCentr_n</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>gui/gui_mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
